tl-tl/sc/getting-started-python.md

```diff
@@ -1,27 +1,27 @@
-# Getting Started with neo-python
+# Pagsisimula gamit ang neo-python
 
 *Main author: [@localhuman](https://github.com/localhuman)*
 
-neo-python, the python node and SDK for the NEO blockchain, enables you to build, test, and run smart contracts on the Neo blockchain. It provides the following main functionalities:
+Ang neo-python, ang python node at SDK para sa NEO blockchain, ay nagbibigay sa iyo ng kakayahan na mag-build, mag-test, at magpatakbo ng mga smart na kontrata sa Neo blockchain. Ito ay nagbibigay ng mga sumusunod na pangunahing functionalidad:
```

Comment:
you could just retain the word `functionality` here

Reply:
fix provided